drizzlepac/hapmultisequencer.py

```diff
@@ -373,7 +378,10 @@ def run_mvm_processing(input_filename, skip_gaia_alignment=True, diagnostic_mode
             # Compute mask keywords early in processing for use in determining what
             # parameters need to be used for processing.
             filter_item.generate_footprint_mask(save_mask=False)
-
+            if not filter_item.mask_computed:
+                log.warning(f"Ignoring {filter_item.info} as no input exposures overlap that layer.")
+                filter_item.valid_product = False
```

Comment:
Maybe I have my objects confused, but it looks like the valid_product attribute is already set in the generate_footprint_mask() method?

Reply:
Good catch!  That is a leftover from all the iterations during development on where this flag should be set.  In fact, the if statement should have been updated to use the flag set by the generate_footprint_mask() method instead of setting that flag here.  **This has been fixed in dbd64d1.**